worker/pricetable.go

```diff
@@ -0,0 +1,156 @@
+package worker
+
+import (
+	"context"
+	"errors"
+	"math"
+	"sync"
+	"time"
+
+	"go.sia.tech/renterd/internal/consensus"
+	"go.sia.tech/renterd/rhp/v3"
+	rhpv3 "go.sia.tech/renterd/rhp/v3"
+	"go.sia.tech/siad/types"
+)
+
+type priceTables struct {
```

Comment:
I'm not 100% sure this warrants its own struct and mutex. If you change the name to `UpdatePriceTable` and make it a method on the worker you can drop this struct and also the back ref to the `worker`. I would just add a `pricetables` map on the worker for the time being

Reply:
I do like to have more simpler objects than a few monolithic ones like the `Renter` and other massive types we had back in siad. It nicely separates concerns and reduces lock contention on the single worker lock.

Reply:
ok - the worker doesn't have a mutex lock yet and I just wanted to get rid of the worker ref. but it's totally fine 👍